client/data/blogging-prompt/use-blogging-prompt.ts

```diff
@@ -5,34 +5,45 @@ import wp from 'calypso/lib/wp';
 export interface BloggingPrompt {
 	id: string;
 	text: string;
+	answered: boolean;
+	answered_users_count: number;
+	answered_users_sample: [ string ];
 }
 
-const selectPrompt = ( response: any ): BloggingPrompt | null => {
-	const prompt = response && response.prompts && response.prompts[ 0 ];
-	if ( ! prompt ) {
+const selectPrompts = ( response: any ): [ BloggingPrompt ] | null => {
```

Comment:
I think you mean `BloggingPrompt[]`? ([an array](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#arrays) of BloggingPrompt objects)

Reply:
Yeah, I wasn't sure what the syntax here was but that's what I was looking for. Thanks.